tests/std/tests/P0220R1_searchers/test.cpp

```diff
@@ -437,6 +437,70 @@ void test_case_randomized_cases() {
     }
 }
 
+#ifndef _M_CEE // TRANSITION, VSO-1659496
+template <class Tag, class T>
+struct tagged_hash {
+    std::size_t operator()(const T& t) const {
+        return std::hash<T>{}(t);
+    }
+};
+
+template <class Tag>
+struct tagged_equal {
+    template <class T, class U>
+    constexpr auto operator()(T&& t, U&& u) const -> decltype(std::forward<T>(t) == std::forward<U>(u)) {
+        return std::forward<T>(t) == std::forward<U>(u);
+    }
+};
+
+template <class T>
+struct holder {
+    T t;
+};
+
+struct incomplete;
+
+template <class T>
+using validating_hash = tagged_hash<holder<incomplete>, T>;
+
+using validating_equal = tagged_equal<holder<incomplete>>;
+
+void test_adl_proof_default_searcher_on_iterators() {

```

Comment:
These should have COMPILE-ONLY comments, as they occur in an otherwise runtime test.